themes/quansightlabs/assets/css/custom.css

```diff
@@ -4484,6 +4500,7 @@ select[multiple].input-group-sm > .input-group-btn > .btn {
     float: none;
     width: auto;
     margin-top: 0;
+    text-align: center;
```

Comment:
I think this to center the elements of the navbar in small screens, right? However, testing seems like is not affecting the alignment of the elements.

To achieve centering the elements of the navbar in small screens we could add something like this in the CSS:
```
#bs-navbar{
  text-align: center;
}
```

Checking with Firefox before and after:

### Before
![image](https://user-images.githubusercontent.com/16781833/79786950-d59df800-830b-11ea-845d-fb03424ae6ed.png)


### After
![image](https://user-images.githubusercontent.com/16781833/79787007-ea7a8b80-830b-11ea-96a0-628a9ad001ab.png)